dm/ctl/master/source_table_schema.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"github.com/pingcap/dm/dm/ctl/common"
+	"github.com/pingcap/dm/dm/pb"
+
+	"github.com/spf13/cobra"
+)
+
+// NewSourceTableSchemaCmd creates a SourceTableSchema command.
+func NewSourceTableSchemaCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "source-table-schema <task-name> <table-filter1> <table-filter2> ...",
```

Comment:
very long command name 😂

Reply:
will `binlog-schema` be renamed to in this PR or next one?

Reply:
in this PR